packages/synapse-react-client/src/components/QueryWrapperErrorBoundary.tsx

```diff
@@ -22,8 +26,8 @@ export function QueryWrapperErrorBoundary({
     [getCurrentQueryRequest],
   )
   const { accessToken } = useSynapseContext()
+  const { clearSession } = useApplicationSessionContext()
```

Comment:
This is going to break in SWC where there is no wrapping `ApplicationSessionContext`

Reply:
Let's see if the polling fixes this issue (as you hypothesized here: https://sagebionetworks.jira.com/browse/PORTALS-3190?focusedCommentId=223776)